.github/workflows/ci_cd.yml

```diff
@@ -79,14 +79,29 @@ jobs:
         run: |
           python -m pip install --upgrade .[doc] tox
 
-      - name: Generate the documentation with tox
-        run: tox -e doc
+      - name: Generate the HTML documentation
+        run: make -c doc html
```

Comment:
@SMoraisAnsys this now looks identical with `ansys/actions/doc-build`. could you convert the `doc-build` step to use that instead?